app/containers/Vaults/index.js

```diff
@@ -468,7 +468,15 @@ const VaultsWrapper = (props) => {
   const migratedVaultsWithBalance = [];
   if (isMigrated) {
     vaultItems.forEach((vault) => {
-      if (vault && vault.balanceOf && vault.balanceOf[0].value > 0) {
+      let decimals = 18;
```

Comment:
@patcito would prefer we don't use any hard-coded decimal values and instead rely on USDC normalized deposit values. If you don't have this already you can get it from `0x83d95e0D5f402511dB06817Aff3f9eA88224B030` `getNormalizedValueUsdc(token, amount)`